main.py

```
@@ -1,243 +1,662 @@
     def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+        # Initialize intents before super().__init__
+        intents = discord.Intents.default()
+        intents.guilds = True
+        intents.messages = True
+        
+        # Pass intents to super().__init__
+        super().__init__(
+            command_prefix="!",
+            intents=intents,
+            *args, 
```

Ruff

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)